data/json/npcs/epilogues.json

```diff
@@ -156,6 +156,174 @@
     "id": "npc_male_27",
     "text": "     Was detained and handcuffed by a police robot on multiple counts of looting and vandalism that were tracked and recorded by the few remaining security systems.  While being held on the ground awaiting additional police response units he was torn to pieces by the undead that were attracted by the commotion."
   },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_28",
+    "text": "     Became a hunter after your death, living alone in the woods.  Traced back to his cabin, he was murdered and devoured in his sleep by an actual cannibal."
```

Comment:
Shia strikes again!

Reply:
Yeah, he's always been one of my favourite monsters!